i18n/messages/pt-BR.json

```
@@ -202,6 +383,356 @@
+        "prescribe": "Prescrever",
+        "edit": "Editar",
+        "delete": "Excluir",
+        "preview": "Pré-visualizar"
+      }
+    }
+  },
+  "privacy": {
+    "title": "Privacidade e Proteção de Dados",
+    "subtitle": "Gerencie seus dados e preferências de privacidade conforme LGPD/GDPR",
+    "loading": "Carregando configurações de privacidade...",
+    "loadError": "Erro ao carregar configurações de privacidade. Tente recarregar a página.",
```

Biome (2.1.2)

[error] The key privacy was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)